node.gyp

```diff
@@ -927,8 +927,16 @@
     ['OS=="aix"', {
       'targets': [
         {
-          'target_name': 'node',
-          'type': 'executable',
+          'conditions': [
+            ['node_shared=="true"', {
+              'target_name': 'libnode.<(node_module_version)',
```

Comment:
That's a weird target name.  What's more, doesn't this belong in the `<(node_core_target_name)` target?